crud/select/compat/select.lua

```diff
@@ -252,15 +253,18 @@ local function select_module_call_xc(space_name, user_conditions, opts)
     checks('string', '?table', {
         after = '?table|cdata',
         first = '?number',
-        timeout = '?number',
         batch_size = '?number',
         bucket_id = '?number|cdata',
         force_map_call = '?boolean',
         fields = '?table',
+        fullscan = '?boolean',
```

Comment:
Please, add the new option into README.

Reply:
Done + I have updated examples from the file.